src/client/scm_estimator/common/tests/scm_estimator.sh

```diff
@@ -0,0 +1,91 @@
+#!/bin/bash
+
+SL_PREFIX="${1}"
+TEST_DIR=$(mktemp -d)
+CURRENT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+export PYTHONPATH="${SL_PREFIX}/lib64/python2.7/site-packages"
```

Comment:
Two comments

1. This doesn't work when python3 is the default as on my Fedora 32 system:

jvolivie-desk1:[~/daos]$ python --version
Python 3.8.3

I don't even have python2.7 as it's been EOLd

So we shouldn't hard code the python version but perhaps discover it and use the appropriate setting

2. I wonder if the setting of PYTHONPATH belongs here or should be moved to setup_local.sh.   Regardless, it should prepend the path rather than replacing it in case the user has their own PYTHONPATH entries for whatever reason.

Reply:
I made some changes and I tested it on Ubuntu 20.04 without python2 installed on it